gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -680,6 +680,120 @@ gst_tensor_filter_common_free_property (GstTensorFilterPrivate * priv)
   gst_tensors_info_free (&priv->out_config.info);
 }
 
+/**
+ * @brief Get available framework from given file when user selects auto option
+ * @param[in] model_files the prediction model paths
+ * @param[in] num_models the number of model files
+ * @return Automatically detected framework name
+ */
+gchar *
+gst_tensor_filter_framework_auto_detection (const gchar ** model_files,
+    unsigned int num_models)
+{
+  const GstTensorFilterFramework *fw = NULL;
+  gchar *file_extension, *file_extension_2;
+  gchar *fw_priority_str = NULL, *detected_fw = NULL;
+  gchar **fw_priority_str_arr;
+  guint i;
+
+  file_extension = strrchr (model_files[0], '.');
+
+  /* Detect framework based on file extension */
+  if (g_strcmp0 (file_extension, ".pb") == 0)
```

Comment:
consider to use g_ascii_strcasecmp instead for ignoring the case of file ext.

Reply:
Thanks! It seems to be less sensitive to the case. Updated.